Lil-Peep/src/styles/modules/site-collection.scss

```diff
@@ -77,6 +77,7 @@
 
     .home-route-section {
       color: #{$home-route-section};
```

Comment:
can you remove `#{}` from variables, please?

Reply:
Got it